client/src/ChartBuilder.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from 'react';
+import { InputText } from 'primereact/inputtext';
+import { Button } from 'primereact/button';
+import ReactECharts from 'echarts-for-react';
+import axios from 'axios';
+
+function ChartBuilder() {
+    const [seriesIdInput, setSeriesIdInput] = useState('');
+    const [charts, setCharts] = useState([]);
+    const [loaded, setLoaded] = useState(false); // <-- ADD THIS
```

Comment:
```suggestion
    const [loaded, setLoaded] = useState(false); 
```
please remove ai code comments